cg/apps/bam_to_cram.py

```diff
@@ -0,0 +1,86 @@
+"""
+    Module for compressing bam to cram 
+"""
+
+import logging
+import tempfile
+from pathlib import Path
+
+from cg.utils import Process
+
+LOG = logging.getLogger(__name__)
+
+
+REFERENCE_PATH = "path/to/reference"
```

Comment:
Path to genome reference hardcoded here

Reply:
read from config?

Reply:
resolution?

Reply:
Yes, the path to the reference to use needs to be given in the cg config now!